plugins/masterdata_cockpit/app/views/masterdata_cockpit/project_masterdata/_masterdata_details.html.haml

```diff
@@ -134,8 +134,8 @@
         = mail_to @project_masterdata.responsible_primary_contact_email
       - else
         not set 
-    .col-sm-1.details-value 
-      = render_propagation_status(@project_masterdata.responsible_primary_propagation_type)
+    -#.col-sm-1.details-value 
```

Comment:
We may need to ensure the row adds up to 12 columns as per the Bootstrap grid system:  
[Bootstrap Grid Options](https://getbootstrap.com/docs/3.3/css/#grid-options)

Currently: `col-sm-4 + col-sm-2 + col-sm-5 = 11`.

To prevent unexpected behavior, I recommend extending one of the columns. For example, you could update `.col-sm-5.details-value` to `.col-sm-6.details-value` to balance the layout properly.